client/state/sharing/publicize/actions.js

```diff
@@ -31,18 +31,18 @@ export function dismissShareConfirmation( siteId, postId ) {
 	};
 }
 
-export function sharePost( siteId, postId, skippedConnections, message ) {
+export function sharePost( siteId, postId, message, connectionIds ) {
 	return ( dispatch ) => {
 		dispatch( {
 			type: PUBLICIZE_SHARE,
 			siteId,
 			postId,
-			skippedConnections,
-			message
+			message,
+			connectionIds,
 		} );
 
 		return new Promise( ( resolve ) => {
```

Comment:
Promise here is not needed. The undocumented method is already returning a Promise.

```
return this.wpcom.req.post( { path: `/sites/${ siteId }/post/${ postId }/publicize`, body, apiVersion: '1.1' }, fn );
```